enhancements/installer/nutanix-ahv-ipi.md

```diff
@@ -0,0 +1,167 @@
+---
+title: nutanix-ahv-ipi
+authors:
+  - "@vnephologist"
+reviewers:
+  - "@makentenza"
+  - "@fabianofranz"
+approvers:
+  - "@makentenza"
+  - "@fabianofranz"
+creation-date: 2021-09-15
+last-updated: 2021-09-24
+status: provisional
+---
+
+# OpenShift Installer Provisioned Infrastructure (IPI) on Nutanix AOS (AHV)
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This enhancement proposes adding support for OpenShift 4 Installer Provisioned Infrastructure (IPI) on Nutanix AOS infrastructure with Nutanix AHV hypervisor. It describes the necessary enhancements, tooling and documentation to enable this capability.
+
+## Motivation
+
+Red Hat and Nutanix have recently [announced](https://www.redhat.com/en/about/press-releases/red-hat-and-nutanix-announce-strategic-partnership-deliver-open-hybrid-multicloud-solutions) a strategic partnership to deliver open hybrid multicloud solutions. For OpenShift this means that it ​​becomes the preferred enterprise full-stack Kubernetes solution on Nutanix Cloud Platform with AHV.
+
+Today, OpenShift can be installed on the Nutanix AOS platform using the OpenShift platform [agnostic installer](https://github.com/nutanix/openshift/tree/main/docs/install/manual), but mutual customers have also requested a fully automated, integrated installer built using OpenShift 4's infrastructure level automation capabilities. This is particularly appealing for customers deploying in "hybrid cloud" scenarios where OpenShift should be deployed using common mechanisms across different clouds.
+
+### Goals
+
+The primary goal of Nutanix AHV IPI is to provide users with an easier path to running OpenShift 4 on Nutanix AOS with AHV.
+
+To achieve that goal, we will:
+- Enhance the installer, adding Nutanix AOS options.
+- Prepare Nutanix infrastructure using the [Nutanix Terraform (TF) provider](https://github.com/nutanix/terraform-provider-nutanix) for provisioning.
+- Integrate with the Nutanix cluster API provider [to-do] and enhance requisite operators to enable cluster functionality.
```

Comment:
When doing this integration, please consider how certificate trust bundles will be provided to the cluster API provider and the in-cluster operators when the Nutanix endpoints are secured with internal CAs.